pkg/operator/staticpod/controller/backingresource/backing_resource_controller_test.go

```diff
@@ -119,7 +120,7 @@ func TestBackingResourceController(t *testing.T) {
 			),
 			validateActions: func(t *testing.T, actions []clienttesting.Action) {
 				createdObjects := filterCreateActions(actions)
-				if createdObjectCount := len(createdObjects); createdObjectCount != 1 {
+				if createdObjectCount := len(createdObjects); createdObjectCount != 2 {
```

Comment:
would prefer to make the tests of the logic and the events orthogonal. This way this gets more and more complicated.

Reply:
i have fake recorder now, so this is not needed, good catch